app/controllers/application_controller.rb

```diff
@@ -1,13 +1,13 @@
 class ApplicationController < ActionController::Base
```

Comment:
why do we need to keep `ApplicationController`?

Reply:
Because ActiveAdmin and Devise (for admin users) use it.

Reply:
https://github.com/lynndylanhurley/devise_token_auth/blob/7f8a0301191c4a857c94a5fcb2029de04bcf0d00/lib/generators/devise_token_auth/install_generator_helpers.rb#L23 this looks like it should inherit from `ActionController::API` but I guess that'd break ActiveAdmin

Reply:
I'll look into it